input/cql/COVID19EmergencyDeptSummary.cql

```diff
@@ -72,12 +72,55 @@ define DispositionSummary: {
 }
 
 // Value: true and false, null
-define TreatmentSummary: {
-    NonPharmacologic: "Recommend Non-Pharmacologic Treatment",
-    Discharge: "Recommend Discharge Treatment",
-    Admission: "Recommend Admission Treatment",
-    Steroids: "Recommend Steroids Treatment"
-}
+define TreatmentSummary: 
+  if ( C19."Is Severe Severity" ) then  
+    {
+    {
+      TreatmentType: 'NonPharmacologic',
+      DisplayOrder: 0,
```

Comment:
#nofix .  seems like a mix of presentation and business to have display order here. We could present natural order but make it the order we want.. and if the presentation really needed to change the order, then would have to use an index key from an exported list of possible treatment keys. Likely not worth the effort though :)